test/utils/utils.test.js

```diff
@@ -537,15 +539,16 @@ describe('Utils', () => {
         });
 
         anchors.forEach((a, index) => {
-          const expectedDomain = Object.values(domainsMap)[index];
+          const expectedDomain = Object.values(domainsMap)[index]?.to
```

Comment:
Is the `.to` value is relevant given the changes?

Reply:
No, it got left behind. Thank you for pointing out 👍